src/allmydata/crypto/aes.py

```diff
@@ -0,0 +1,90 @@
+import six
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import (
+    Cipher,
+    algorithms,
+    modes,
+    CipherContext,
+)
+
+
+DEFAULT_IV = '\x00' * 16
+
+
+def create_encryptor(key, iv=None):
+    """
+    Create and return a new object which can do AES encryptions with
+    the given key and initialization vector (IV). The default IV is 16
+    zero-bytes.
+
+    The returned object is suitable for use with `encrypt_data`
```

Comment:
Maybe a marker interface would be a clearer way to document this?  `IEncryptor` with no methods, returned by this function and accepted by the other function.

Reply:
Hmm, okay. I sort of started down this road at some point but how I do make "whatever this returns" return *be* an instance of `IEncryptor` or so without a wrapper-class? `zope.interface.directlyProvides()`...?

Reply:
Yea, probably that.  I guess another option is to do something like:

Reply:
Okay I did `IEncryptor` + `IDecryptor`